### ISSUE TITLE
[Suggestion] [Potential Spoiler] - Pen Names; Add "Don't use a Pen Name" option.

### ISSUE THREAD

**Issue by Ghost129er** [Monika-After-Story/MonikaModDev#4825]
**- Pretend this is an unseen topic that Monika is bringing up -**
(tl;dr at bottom)

---------------------------------------------------------------------------------------------------

She starts talking about Pet Names and asks if you have/go by a Pen Name and you have two options:
- Yes
- No

No is obvious what happens, the dialogue continues.
When you choose YES, you type your Pen Name (bob) and proceed to seeing the dialogue. 

When you repeat this topic, she asks you, do you still go by that name or what, you have two options:
- Yes
- No

Yes, again obvious what happens.
No leads her to asking if you, the player, want to tell her your new Pen Name, or absolutely not. 

Now again, two things; 
Choosing yes carries on with the currently/previously entered string/name and the dialogue.
Choosing NO goes with her being somewhat down, BUT when you return and repeat this topic again, your Pen Name is still Bob. HOWEVER, if you make it blank, she says you can type 'nevermind' to chicken out and your name is still 'bob' instead of it being removed.

-----------------------------------------------------------------------------------------------

**tl;dr** - add a "_I'm no longer going by a Pen Name_" option to the **Pen Names** topic choices after having chosen one and when the topic gets repeated.

**Comment by ThePotatoGuy** [comment]
[closing until we look into it]
